sidecar/proxy/envoy/manager.go

```diff
@@ -405,7 +404,7 @@ func buildRoutes(ruleList []api.Rule) []Route {
 
 				runtime := &Runtime{
 					Key:     buildWeightKey(backend.Name, backend.Tags),
-					Default: 0,
+					Default: int(backend.Weight * 100),
```

Comment:
hmm.. Okay, now that sounds like a bad idea.. Lets stick to this current format.